### ISSUE TITLE
Add better font for the theme

### ISSUE THREAD

**Issue by pourmand1376** [rohandebsarkar/academic-folio#69]

**Comment by rohandebsarkar** [comment]
I actually have a strong opinion on this. I believe we should leave it to the website viewers to choose their own fonts rather than downloading a specific font every time someone views the website.

But if you feel this feature maybe necessary for many users then we can think of a way to provide the users to choose their own fonts.

And I would also like to keep the resource calls as minimum as possible for a new visitor. If I am not wrong presently we make two resource calls (one for the html and one for the css), except on pages with math typesetting (then katex dependencies are required, I will look into it later if we can include katex dependencies within html during build time so we do not have to depend on CDNs).

**Comment by rohandebsarkar** [comment]
> This is what al-folio uses.
> 
>     * https://fonts.googleapis.com/css?family=Roboto:300,400,500,700%7CRoboto+Slab:100,300,400,500,700%7CMaterial+Icons
> 
> 
> ```
> <link rel="stylesheet" type="text/css" href="https://fonts.googleapis.com/css?family=Roboto:300,400,500,700%7CRoboto+Slab:100,300,400,500,700%7CMaterial+Icons">
> ```

This was included as a part of material bootstrap. While writing code for https://github.com/alshedivat/al-folio/pull/741, I felt it was quite unnecessary if users do not want to use material design. I included a config to disable material design which will not include the font in al-folio.

**Comment by pourmand1376** [comment]
I agree with your point. I think we should give the user the ability to change the font if he/she wants. 

Today, I wanted to change the font to something else and I put in an hour and still I couldn't do it.
